src/Cake.Common/Tools/GitReleaseNotes/GitReleaseNotesAliases.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.GitReleaseNotes
+{
+    using global::Cake.Core.IO;
```

Comment:
Yes :-( ReSharper is currently fighting me, and I haven't sat down and looked into it yet.  I have ReSharper set up with all the default rules, including putting usings with the namespaces, which isn't what cake does.  As a result, each time I have to remember to put them back properly.  I am starting to move away from that rule in general though, and I need to turn off that refactoring within ReSharper, I just need to find where it is :smile: 

I also found another instance of `global::` within one of the files for GitReleaseManager, and although not related to this PR, I took the opportunity to change it just now in this PR.  Let me know if you want me to back that out.